doc/command_ref.rst

```diff
@@ -689,14 +689,17 @@ transactions and act according to this information (assuming the
 
 .. _history_list_command-label:
 
-``dnf history [list] [<spec>...]``
+``dnf history [list] [--reverse] [<spec>...]``
     The default history action is listing information about given transactions
     in a table. Each ``<spec>`` can be either a ``<transaction-spec>``, which
     specifies a transaction directly, or a ``<transaction-spec>..<transaction-spec>``,
     which specifies a range of transactions, or a ``<package-name-spec>``,
     which specifies a transaction by a package which it manipulated. When no
     transaction is specified, list all known transactions.
 
+``-r``, ``--reverse``
+    The order of `history list` output is printed in reverse order.
+
```

Comment:
I couldn't find a similar situation throughout the man page, but I think it would be good to indent this paragraph by 4 spaces to put it under the `history list` command hierarchically. I've tested it and it gets correctly formatted in both html and man.

Reply:
I indented bot the title and the paragraph. Is this what you mean?

Reply:
Yes. One more thing, `history list` should be in double backticks, the single backticks don't behave the same, I forgot the exact difference...

Reply:
ok, there you go!